drivers/hmis/app/graphql/types/hmis_schema/disability_group.rb

```diff
@@ -35,6 +35,7 @@ class HmisSchema::DisabilityGroup < Types::BaseObject
     field :t_cell_source, HmisSchema::Enums::Hud::TCellSourceViralLoadSource, null: true
     field :viral_load_available, HmisSchema::Enums::Hud::ViralLoadAvailable, null: true
     field :viral_load, Integer, null: true
+    field :viral_load_source, HmisSchema::Enums::Hud::TCellSourceViralLoadSource, null: true
```

Comment:
Bug, I think: Viral load source is not correctly collected. Even when "Medical report" is selected and then the intake is saved, the disability record's viral load source field is still set to 0.

Fix implemented here: Add the viral load source field to the disability object schema, so that the processor knows what enum in to use when we call `attribute_value_for_enum(graphql_enum(field), value)`.

Outstanding question: If we confirm this is a bug, we probably need to backfill data based on the raw form values stored in the processor. Let's create a separate ticket for that work to track it separately?

Reply:
Yeesh good find! Created https://github.com/open-path/Green-River/issues/6864

Reply:
Sounds good, I created #4840 and reverted the change here to prevent merge conflict. This PR's build will fail until that one is merged into release-138, I think.

Reply:
(interestingly the test failures I'm seeing on CI are not the same as the failure I would expect based on the above, so I'm retriggering it to see if that's something to investigate)

Reply:
On the test rerun I'm seeing the failures I expect to see. We should still make sure everything goes green once the other PRs are merged, but I think there is nothing else to investigate here